docs/configuration_file.md

```diff
@@ -59,6 +59,22 @@ Some options are also mandatory.
 
 * `koji_kerberos_principal` (*optional*, `string`) - kerberos principal for the keytab provided in `koji_kerberos_keytab`
 
+* `flatpak` (*optional*, `boolean`) - If true, will build a Flatpak as an OCI image rather than a docker image
+
+* `module` (*optional*, `string`) - Module to build the image against, in the form `NAME:STREAM[:VERSION]`. Currently used only for flatpaks.
```

Comment:
Shouldn't this be a per-build parameter rather than something in a configuration file in the build system?